contrib-03-AnalysisWithPython/Readme.md

```diff
@@ -0,0 +1,153 @@
+also dedicated to data analysis are now very mature with a much larger
+users community.
+The goal of this project was to build an analysis demonstrator capable of :
+- use the rich environment of python libraries to perform analysis tasks
+- load and store processed data in ROOT file format, in order to be able to
+  fully collaborate with ROOT users.
+
+## Environment setup
+
+In order to install and load all the packages we need for this project, we rely
+on the [anaconda python distribution](https://www.continuum.io/downloads).
+
+We tested the following environment on lxplus :
+
+- Install miniconda (or anaconda) for python2.7, Linux 64 bit
+  [link](http://conda.pydata.org/miniconda.html)
+- A non-mandatory step would be to create a virtual environment to keep the
+  local python distribution as clean as possible. It can be achieve trough
+  : `conda create -n NameOfYourVirtualEnv python=2.7 anaconda`.
+  To activate it, do `source activate NameOfYourVirtualEnv` and `source deactivate` to go back to normal. 
+- Install ROOT inside the conda environment : `conda install -c chrisburr root=6.06.02`
+- You will then need to activate the conda environment :
+    - `source activate root` to load the main conda environment
+    - `source activate NameOfYourVirtualEnv` if you made it yourself
+- A few packages are also needed and can be installed using pip, the python
+  package manager :
+    - Numpy (arrays and tools for scientific computing), Matplotlib (plotting
+      library), scikit-learn (Machine Learning in Python) : `pip install numpy matplotlib scikit-learn`
+    - The [root_pandas package](https://github.com/ibab/root_pandas) is based on   [root_numpy](https://github.com/rootpy/root_numpy) and allows you to load a ROOT
+      file and store the content in a pandas dataframe. Follow both links if
+      you want more information on how to load a ROOT file. To install it, do
+      `pip install root_pandas`
+    - In order to produce a jupyter notebook which is a way to keep both the
+      analysis code and its output in the same file, you can also do `pip install jupyter`. Then, to load a jupyter notebook, you can do
+      `jupyter notebook` and access the notebook in your browser.
+
+## Loading data
+
+Using the root_pandas library, we can load a ROOT file in a few simple steps :
+- choose the file you want and store the path as string : `myfile = "path/to/file.root"`
+- then select the branches you want by creating a list with the names of the
+  variables you want. The use of of the wildcard `*` is also supported. For
+  instance you can select `mybranches = ["B0_M","*_PT"]` to load the B0 mass
+  and the transverse momentum of each particle of your entry.
+- As the file will be loaded in the memory, you should limit to a certain
+  number of entries and set, for instance, `n_events = 10000`.
+- Finally, select the name of your tuple as `tuple_name = 'someDir/myTree`
```

Comment:
Maybe the code corresponding to all these steps could be moved to its own code block after the enumeration with its own comments to help:

```python
# Path of the ROOT file
myfile = "path/to/file.root"
# Branches to load, wildcards allowed
mybranches = ["B0_M", "*_PT"]
# Number of entries to load
n_events = 10000
# Name of the tree
tuple_name = "someDir/myTree"
```